### PR TITLE
Importers: Remove the GoDaddy GoCentral importer

### PR DESCRIPTION
**Pull request by jeroenpf**
It has been concluded that the import is no longer reliable and therefore we remove it as an option from Calypso

#### Changes proposed in this Pull Request

- This removes the GoDaddy importer from the calypso importer list.
- It also removes references to the importer from the importer signup flow (which will only support wix).

#### Testing instructions

– Click on Tools/Import in the sidebar and make sure that the GoDaddy importer is not listed.

### CODE REVIEW

File: client/signup/steps/import-url/index.jsx [hunk 1]
```diff
@@ -97,9 +96,7 @@ class ImportURLStepComponent extends Component {
 				} ) => {
 					if ( ! includes( importerTypes, 'url' ) ) {
 						return this.setUrlError(
-							translate(
-								"That doesn't seem to be a Wix or GoDaddy site. Please check the URL and try again."
-							)
+							translate( "That doesn't seem to be a Wix site. Please check the URL and try again." )
```
Comment (a8ci18n) [inline client/signup/steps/import-url/index.jsx:99]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/signup/steps/import-url/index.jsx [hunk 2]
```diff
@@ -289,7 +285,7 @@ class ImportURLStepComponent extends Component {
 
 		const headerText = translate( 'Where can we find your old site?' );
 		const subHeaderText = translate(
-			'Enter your Wix or GoDaddy GoCentral site URL, sometimes called a domain name or site address.'
+			'Enter your Wix site URL, sometimes called a domain name or site address.'
```
Comment (a8ci18n) [inline client/signup/steps/import-url/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Enter your Wix site's URL, sometimes called a domain name or site address.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-godaddy-gocentral-importer)

Comment (yoavf) [reply client/signup/steps/import-url/index.jsx]: Let's do this

**Review by yoavf — APPROVED** [review]
Looks good to me. 
Also tested the import flow `start/import` to make sure that Wix imports still work as expected.

**Pull request merged**